web/src/Web.App/Views/SchoolSpending/Index.cshtml

```diff
@@ -38,12 +38,22 @@
 
     @foreach (var category in Model.PriorityCosts)
     {
+        var categoryToLower = @category.Rating.Category == "Other costs"
```

Comment:
There are constants in `Web.App.Domain.Category` that could be used here instead

Reply:
thanks, fixed in latest commit